src/internal/mediaUtil.ts

```diff
@@ -74,3 +74,25 @@ export function validateViewImagesInput(uriList: media.ImageUri[]): boolean {
   }
   return true;
 }
+
+/**
+ * Returns true if the scan barcode param is valid and false otherwise
+ */
+export function validateScanBarCodeInput(barCodeConfig: media.BarCodeConfig): boolean {
+  if (barCodeConfig != null) {
+    if (
+      barCodeConfig.barCodeType != null &&
```

Comment:
null and undefined check comment is applicable here and for timeOutIntervalInSec as well

Reply:
done